composer.json

```diff
@@ -6,7 +6,7 @@
     "homepage": "https://github.com/griidc/pelagos/",
     "license": "BSD-2-Clause",
     "require": {
-        "php": "^8.0",
+        "php": "^8.1",
```

Comment:
This should just be removed, there is no need for it. But we are going to do it later.